config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  get '/pizzerias/new', to: 'pizzerias#new', as: 'new_pizzeria'
-  post '/pizzerias', to: 'pizzerias#create', as: 'pizzerias'
-  get '/pizzerias/:id', to: 'pizzerias#show', as: 'pizzeria'
-  get '/pizzerias', to: 'pizzerias#index'
+  resources :pizzerias, only: [:new, :create, :show, :index] do
+    resources :pizzas, only: [:new, :create, :show]
+  end
+
```

Comment:
Nice change, but watch the SPACING!
```suggestion
    resources :pizzerias, only: [:new, :create, :show, :index] do
    resources :pizzas, only: [:new, :create, :show]
  end
```